README.md

```diff
@@ -5,24 +5,19 @@ Can be used in combination with [Release Drafter](https://github.com/toolmantim/
 
 ## Usage
 
-Add `.github/main.workflow` with the following:
+Add `.github/workflows/pr-labeler.yml` with the following:
 
-```
-workflow "Add label to PR" {
-  on = "pull_request"
-  resolves = "PR Labeler"
-}
-
-action "PR opened filter" {
-  uses = "actions/bin/filter@master"
-  args = "action opened"
-}
-
-action "PR Labeler" {
-  needs = "PR opened filter"
-  uses = "TimonVS/pr-labeler@master"
-  secrets = ["GITHUB_TOKEN"]
-}
+```yml
+name: Label PRs
+on: [pull_request]
+jobs:
+  pr-labeler:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: TimonVS/pr-labeler@master
```

Comment:
I forgot to change this, but it should be:

```suggestion
      - uses: TimonVS/pr-labeler-action@master
```

The former still works because the repository was renamed from pr-labeler to pr-labeler-action.